CMakeLists.txt

```diff
@@ -81,12 +66,19 @@ enable_testing()
 target_include_directories (unittest PUBLIC test)
 target_include_directories (benchmarks PUBLIC test)
 
-if (ENABLE_AVX)
-    target_compile_definitions (qrack PUBLIC ENABLE_AVX=1)
-    set(QRACK_COMPILE_OPTS -mavx)
-else (ENABLE_AVX)
-    target_compile_definitions (qrack PUBLIC ENABLE_AVX=0)
-endif (ENABLE_AVX)
+if (ENABLE_COMPLEX8)
```

Comment:
Where does ENABLE_COMPLEX8 get declared as an option?

Reply:
By default, we want it off. It becomes active if it's specified on command line. I think double accuracy should be default, but this optionally enables single accuracy. It gives us an extra qubit, but it's also needed for some (or all) current integrated graphics units.